src/structures/GuildMember.js

```diff
@@ -136,6 +145,26 @@ class GuildMember extends Base {
     return this.guild.voiceStates.cache.get(this.id) || new VoiceState(this.guild, { user_id: this.id });
   }
 
+  /**
+   * A link to the member's server avatar.
+   * @param {ImageURLOptions} [options={}] Options for the Image URL
+   * @returns {?string}
+   */
+  avatarURL({ format, size, dynamic } = {}) {
+    if (!this.avatar) return null;
+    return this.client.rest.cdn.Avatar(this.id, this.avatar, format, size, dynamic);
```

Comment:
This won't work, server avatars use a different URL, which you'll need to add to `src/util/Constants.js`

`/guilds/:guildID/users/:userID/avatars/:hash`

Reply:
You're absolutely right, I missed this. Can you give me a guildId, a userId and a hash so I can test?

Reply:
https://cdn.discordapp.com/guilds/692925064135311390/users/90339695967350784/avatars/67fcd6af88d76888b8525c5c3684b6b2.png?size=2048